### ISSUE TITLE
Automatic color conversion

### ISSUE THREAD

**Issue by andraz** [Jackardios/vscode-css-to-tailwindcss#1]
Could this extension also prettify the colors by finding the closest standard tailwind color?

Example: https://find-nearest-tailwind-colour.netlify.app/

**Comment by andraz** [comment]
https://github.com/edjw/find-nearest-tailwind-colour

**Comment by Jackardios** [comment]
@andraz Good idea! At the moment, the extension compares colors strictly, but regardless of the color format (e.g. `#123456` == `rgb(18, 52, 86)`)
